apps/openmw/mwrender/sky.cpp

```diff
@@ -322,11 +321,12 @@ class ModVertexAlphaVisitor : public osg::NodeVisitor
     int mMeshType;
 };
 
-class CelestialBody
+// A base class for the sun and moons. Must be stored in an osg::ref_ptr due to being
+// derived from osg::Referenced.
```

Comment:
Good point, I hadn't seen it used very much in the project, but the wiki does ask that they be used.